docs/bestPractices.md

```diff
@@ -0,0 +1,43 @@
+>
+> &mdash; [Google](https://developers.google.com/web/fundamentals/web-components/best-practices#aim-to-keep-primitive-data-attributes-and-properties-in-sync,-reflecting-from-property-to-attribute,-and-vice-versa.)
+
+
+> We recommend reflecting from an attribute to a property, but to avoid reflecting from properties to attributes. This is because with custom elements properties can update often and triggering a DOM change for each update can impact performance.
+>
+> &mdash; [Open-WC](https://open-wc.org/guides/knowledge/attributes-and-properties/#attribute-and-property-reflection)
+
+### Additional reference
+- [auro-badge PR](https://open-wc.org/guides/knowledge/attributes-and-properties/#attribute-and-property-reflection)
+- [Impact on Svelte](https://css-tricks.com/using-custom-elements-in-svelte/#attributes-used-as-styling-hooks)
+
+## If you query the children of a custom element, react to slot changes
+
+In some custom elements (e.g. auro-accordion-group) we need to keep track of the children in that element's slot. When doing so, make sure you react to [slot changes](https://developer.mozilla.org/en-US/docs/Web/API/HTMLSlotElement/slotchange_event) instead of only initializing once. This allows children to be dynamically added after the first time the element renders. Additionally, not doing this has [caused issues](https://github.com/AlaskaAirlines/auro-accordion/issues/18) in frameworks like Svelte that insert the parent into the DOM before the children.
+
+### Additional reference
+- [auro-accordion PR](https://github.com/AlaskaAirlines/auro-accordion/pull/21)
+
+## Give boolean properties default values
+
+When adding boolean properties to an element, give them a default value. Otherwise, the property will be initialized to `undefined`. This will usually work as expected, since `undefined` is falsy. However, it's more correct to explicitly set the property to `false`.
+
+This has two main benefits:
+- If the property is retrieved from outside the element, it will return either `true` or `false` instead of `undefined`. Native DOM elements behave the same way, i.e. `document.querySelector('button').disabled` returns `false` instead of `undefined` if `disabled` is not set.
+- Svelte will understand that the property is a boolean and allow the consumer to write `<auro-button disabled>`. Otherwise, the consumer has to write `<auro-button disabled={true}>`.
```

Comment:
```suggestion
1. If the property is retrieved from outside the element, it will return either `true` or `false` instead of `undefined`. Native DOM elements behave the same way, i.e. `document.querySelector('button').disabled` returns `false` instead of `undefined` if `disabled` is not set.
1. Svelte will understand that the property is a boolean and allow the consumer to write `<auro-button disabled>`. Otherwise, the consumer has to write `<auro-button disabled={true}>`.
```